ibm_catalog.json

```diff
@@ -0,0 +1,546 @@
+                                    "value": "eu-de"
+                                },
+                                {
+                                    "displayname": "London (eu-gb)",
+                                    "value": "eu-gb"
+                                },
+                                {
+                                    "displayname": "Madrid (eu-es)",
+                                    "value": "eu-es"
+                                },
+                                {
+                                    "displayname": "Dallas (us-south)",
+                                    "value": "us-south"
+                                },
+                                {
+                                    "displayname": "Montreal (ca-mon)",
+                                    "value": "ca-mon"
+                                },
+                                {
+                                    "displayname": "Toronto (ca-tor)",
+                                    "value": "ca-tor"
+                                },
+                                {
+                                    "displayname": "Washington (us-east)",
+                                    "value": "us-east"
+                                },
+                                {
+                                    "displayname": "Sao Paulo (br-sao)",
+                                    "value": "br-sao"
+                                }
+                            ]
+                        },
+                        {
+                            "key": "existing_resource_group_name",
+                            "type": "string",
+                            "default_value": "Default",
+                            "description": "The name of an existing resource group to provision the resources.",
+                            "display_name": "resource_group",
+                            "required": false,
+                            "custom_config": {
+                                "type": "resource_group",
+                                "grouping": "deployment",
+                                "original_grouping": "deployment",
+                                "config_constraints": {
+                                    "identifier": "rg_name"
+                                }
+                            },
+                            "virtual": false
+                        },
+                        {
+                            "key": "cos_instance_name",
+                            "type": "string",
+                            "default_value": "cos-cybervault",
+                            "display_name": "cos_instance_name",
+                            "description": "The name for the IBM Cloud Object Storage instance provisioned by this solution. A default name has been provided. The instance will be named with the prefix plus this value in the format <prefix>-value.",
```

Comment:
avoid duplicating the type, display_name and description in this json file so you don't have to maintain it in 2 places. If not specified it will be pulled from the terraform code. Same applies or all inputs